pkg/gofr/datasource/opentsdb/response.go

```diff
@@ -13,6 +13,12 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+var (
+	errSendingRequest  = errors.New("opentsdb client error")
```

Comment:
```suggestion
	errSendingRequest  = errors.New("OpemTSDB client error")
```

Reply:
every error must start in smallcase. Hence modifying to `openTSDB`.

Reply:
Not exactly

https://go.dev/wiki/CodeReviewComments#error-strings

> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.